scm-plugins/scm-git-plugin/src/main/java/sonia/scm/repository/spi/GitWorkingCopyInitializer.java

```diff
@@ -66,38 +69,13 @@ public SimpleGitWorkdirFactory(WorkdirProvider workdirProvider) {
         throw notFound(entity("Branch", initialBranch).in(context.getRepository()));
       }
 
-      return new ParentAndClone<>(null, clone);
+      return new ParentAndClone<>(null, clone, target);
     } catch (GitAPIException | IOException e) {
       throw new InternalRepositoryException(context.getRepository(), "could not clone working copy of repository", e);
+    } finally {
+      long end = System.nanoTime();
+      long duration = end - start;
+      LOG.trace("took {} ns to clone repository {}", duration, context.getRepository().getId());
```

Comment:
Why the manual calculation and the output in ns? Why not use Stopwatch?

Reply:
Right, thx